schema/config.d.ts

```diff
@@ -0,0 +1,61 @@
+import { Feature, SiteException } from './feature';
+import { AutoconsentFeature } from './features/autoconsent';
+import { CookieFeature } from './features/cookie';
+import { TrackerAllowlistFeature } from './features/tracker-allowlist';
+import { WebCompatFeature } from './features/webcompat';
+import { DuckPlayerFeature } from './features/duckplayer';
+
+export { WebCompatSettings } from './features/webcompat';
+export { DuckPlayerSettings } from './features/duckplayer';
+
+export type ExportedSchemas =
+    | 'GenericV4Config'
+    | 'AndroidV4Config'
+    | 'LegacyConfig'
+    | 'LegacyAndroidConfig'
+    | 'WebCompatSettings'
+    | 'DuckPlayerSettings';
+
+/**
+ * Defines the structure of the built V4 config output as downloaded by clients.
+ */
+export type ConfigV4<VersionType> = {
+    readme: string;
+    version: number;
+    features: Record<string, Feature<any, VersionType>> & {
```

Comment:
The previous one, checked for feature name matching a format. Can you raise a follow up issue to make sure we add a unit test? (Unless there's a way to do this in TS)?

Reply:
Does https://github.com/duckduckgo/privacy-configuration/pull/2439/commits/4e6f8f620699c08b2190301bb817c40961bf9844 cover it?

Reply:
Thanks!